ports/mfx-dispatch/portfile.cmake

```
@@ -1,26 +1,34 @@
 vcpkg_from_github(
-        OUT_SOURCE_PATH SOURCE_PATH
-        REPO lu-zero/mfx_dispatch
-        REF 1.35.1
-        SHA512 12517338342d3e653043a57e290eb9cffd190aede0c3a3948956f1c7f12f0ea859361cf3e534ab066b96b1c211f68409c67ef21fd6d76b68cc31daef541941b0
-        HEAD_REF master
-        PATCHES
-            fix-unresolved-symbol.patch
-            fix-pkgconf.patch
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO lu-zero/mfx_dispatch
+    REF 1.35.1
+    SHA512 12517338342d3e653043a57e290eb9cffd190aede0c3a3948956f1c7f12f0ea859361cf3e534ab066b96b1c211f68409c67ef21fd6d76b68cc31daef541941b0
+    HEAD_REF master
+    PATCHES
+        fix-vcpkg.patch
 )
 
 set(LIBRARY_TYPE ${VCPKG_LIBRARY_LINKAGE})
 if (LIBRARY_TYPE STREQUAL "dynamic")
     set(LIBRARY_TYPE "shared")
 endif()
 
-vcpkg_cmake_configure(
-    SOURCE_PATH "${SOURCE_PATH}" 
-)
+if(VCPKG_CMAKE_SYSTEM_NAME STREQUAL "Linux")
+    # Linux CMakeLists.txt is broken.
+    vcpkg_configure_make(

```

```suggestion
    vcpkg_cmake_configure(
```
Please use the new function.